Script/install.sh

```diff
@@ -60,6 +60,7 @@ display_welcome_msg() {
 	echo "   - python3-smbus    Python3 bindings for Linux SMBus access through i2c-dev"
 	echo "   - arduino          AVR development board IDE and built-in libraries"
 	echo "   - minicom          friendly menu driven serial communication program"
+  echo "   - numpy            is the fundamental package for scientific computing with Python"
```

Comment:
I don't see any bad indentations. I've tried to use spaces and tabs again, but it seems aligned and git doesn't see any differences.
Here's an example:

![good_indentation](https://user-images.githubusercontent.com/26958764/27500996-7a1ee8a6-5873-11e7-8bc7-914090097614.PNG)